pages/03-plural-features/02-k8s-upgrade-assistant/01-upgrade-insights.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Upgrade Insights
+description: How to configure and check Upgrade Insights
+---
+
+Plural integrates insights from specific cloud provider clusters. These cluster insights offer recommendations to 
+help you adhere to best practices for Kubernetes and cloud providers. They also provide information on issues that 
+could prevent upgrades and suggest ways to resolve them.
+
+## Supported providers
+Currently, the following provider is supported:
+
+- Amazon Elastic Kubernetes Service (EKS)
+
+## Configuration
+Upgrade Insights can be configured using our {% doclink to="overview_agent_api_reference" %}Custom Resource Definition (CRD){% /doclink %}. This configuration allows you to provide
+custom credentials, which are used to access the cloud provider's API and gather relevant information. For the EKS
+provider, we automatically configure node-level permissions during the cluster bootstrapping process with Plural,
+enabling our operator to retrieve upgrade insights.
+
+### Relying on Node-Level Permissions
+Alternatively, you can rely on node-level permissions via the cloud provider's assume role feature with 
```

Comment:
add the terraform to set it up here as well (can just copypasta it from the github.com/pluralsh/bootstrap repo)